records/board/mlstl476qe.yaml

```diff
@@ -0,0 +1,11 @@
+common:
+    macros:
+        - IO_CONFIG_OVERRIDE
```

Comment:
Hi @liuxinbai, the define IO_CONFIG_OVERRIDE should be only defined if the pinout in [IO_Config.h](https://github.com/mbedmicro/DAPLink/blob/master/source/hic_hal/nxp/lpc11u35/IO_Config.h) is incompatible with your hardware and you need to supply your own.